api/v1/app.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+"""DOCSTRING"""
+from flask import Flask, jsonify
+from models import storage
+from api.v1.views import app_views
+
+
+app = Flask(__name__)
```

Comment:
Just wanna point out, I think this work was supposed to be on a separate branch. Storage_get_count is gonna be manual reviewed, and I think it was only supposed to have the get() and count() methods on it